packages/cursor/init-gpg.sh

```
@@ -0,0 +1,105 @@
+    for preset_path in \
+        "/usr/lib/gnupg/gpg-preset-passphrase" \
+        "/usr/lib/gnupg2/gpg-preset-passphrase" \
+        "/usr/libexec/gpg-preset-passphrase" \
+        "$(command -v gpg-preset-passphrase 2>/dev/null || echo '')"; do
+        if [[ -x "$preset_path" ]]; then
+            GPG_PRESET="$preset_path"
+            break
+        fi
+    done
+
+    if [[ -z "$GPG_PRESET" ]]; then
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)